app/models/track.rb

```diff
@@ -9,6 +9,9 @@ class Track < ApplicationRecord
 
   belongs_to :song, touch: true
   belongs_to :show
+  belongs_to :previous_track, :class_name => 'Track', optional: true
```

Comment:
just because i'm a syntax nerd -- can you go through the PR and change old style hash (:foo => bar) to use new style (foo: bar).